code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -49,9 +65,11 @@
 /obj/item/weapon/reagent_containers/blood/OMinus
 	blood_type = "O-"
 
+/obj/item/weapon/reagent_containers/blood/lizard
+	blood_type = "L"
```

Comment:
lizards now have a special blood type that only they use. No blood of that type is stored in medbay storage, dunno if we should or not.